src/work-allocation-2/containers/work-case-list/work-case-list.component.spec.ts

```diff
@@ -0,0 +1,459 @@
+import { CdkTableModule } from '@angular/cdk/table';
+import { Component, Input, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { LoadingService, PaginationModule } from '@hmcts/ccd-case-ui-toolkit';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { of } from 'rxjs';
+
+import { WorkCaseListComponent } from './work-case-list.component';
+import { FieldConfig, SortField } from '../../models/common';
+import { CaseService, SortOrder } from '../../enums';
+import { Case, CaseAction, CaseServiceConfig } from '../../models/cases';
+import { PaginationParameter } from '../../models/dtos';
+import { WorkAllocationComponentsModule } from '../../components/work-allocation.components.module';
+import { WorkAllocationCaseService } from '../../services';
+import { getMockCases, MockRouter } from '../../tests/utils.spec';
+import { ConfigConstants } from '../../components/constants';
```

Comment:
_Nitpick:_
These imports need ordering.

Reply:
Sorted.

Reply:
Sorted, with a little help from https://stackoverflow.com/questions/41805723/how-to-order-imports-with-tslints-import-ordering-rule